front-end/src/components/Post/GovernanceSideBar/Referenda/VoteReferendum.tsx

```diff
@@ -129,20 +129,27 @@ const VoteRefrendum = ({ className, referendumId, address, accounts, onAccountCh
 						<Loader text={loadingStatus.message}/>
 					</Card>
 					: <Card>
-						<AccountSelectionForm
-							title='Vote with account'
-							accounts={accounts}
-							address={address}
-							withBalance
-							onAccountChange={onAccountChange}
-						/>
-						<BalanceInput
-							label={'Lock balance'}
-							helpText={'Amount of you are willing to lock for this vote.'}
-							placeholder={'123'}
-							onChange={onBalanceChange}
-						/>
-						<VoteLock/>
+						{apiReady
```

Comment:
checking for `apiReady` and telling "api not available" isn't right. Either it's available, but not ready (and again, it shouldn't take forever) or it's not available.

Why not having a loader here?

Reply:
Looking at it again, the `apiReady` is sort of redundant here. All info needed comes from injected accounts, not the api, and the voting buttons are disabled as long as the api isn't ready (which shouldn't be the case; the card only shows up after the user having clicked on the Vote button - that's plenty of time)